### PR TITLE
Minimal PyTorch test

### PR DESCRIPTION
**Pull request by lugi0**
Signed-off-by: Luca Giorgi <lgiorgi@redhat.com>

### CODE REVIEW

**Comment by lugi0** [comment]
This cannot currently be tested since the spawning of the image is blocked

File: tests/Tests/500__jupyterhub/minimal-pytorch-test.robot [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Library          DebugLibrary
+Library          JupyterLibrary
+Suite Setup      Begin Web Test
+Suite Teardown   End Web Test
+
+*** Variables ***
+
+
+*** Test Cases ***
+Minimal CUDA test
```
Comment (vasukulkarni) [inline tests/Tests/500__jupyterhub/minimal-pytorch-test.robot:15]: Regression tag?

Comment (lugi0) [reply tests/Tests/500__jupyterhub/minimal-pytorch-test.robot]: Added in latest commit

**Comment by lugi0** [comment]
PR test:
https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/98/

**Comment by lugi0** [comment]
Fixed the import error in the PR test.
New PR test:
https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/100/

**Comment by lugi0** [comment]
PR test job: https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/204/

**Comment by lugi0** [comment]
Unexpected failures, rerunning: https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/206/

**Comment by lugi0** [comment]
PR test here: https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/208/
Last run was aborted due to issues with the user's PVC

**Comment by lugi0** [comment]
Rerunning, still issues with the PVC: https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/211/

**Comment by lugi0** [comment]
Pr test: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/150/

Same deal as #69, I'd like to get some reviews here before merging.

@vasukulkarni @jgarciao @bdattoma @pablofelix @aloganat @LaVLaS

**Comment by lugi0** [comment]
Previous run hit some Jenkins config issue, latest run here: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/151

**Comment by vasukulkarni** [comment]
closing in favor of #69

**Pull request closed without merging**